screen-balance/src/main/java/com/ivy/balance/BalanceEvent.kt

```diff
@@ -0,0 +1,9 @@
+package com.ivy.balance
+
+import com.ivy.legacy.data.model.TimePeriod
+
+sealed interface BalanceEvent {
+    data class OnSetPeriod(val timePeriod: TimePeriod) : BalanceEvent
+    object OnPreviousMonth : BalanceEvent
```

Comment:
Make those `data object` cuz Ivy Wallet supports Kotlin 1.9 and we can have a nice `toString` implementation

Reply:
Done